rule-packs/azure.json

```diff
@@ -383,5 +383,16 @@
             "version": "v1"
           }
         ]
-    }
-  ]
\ No newline at end of file
+    },
+    {
+      "name": "azure-virtual-machine-disablePasswordAuthentication",
+      "description": "Finds Azure virtual machines that allow all access with disabledPasswordAuthentication. ",
+      "queries": [
+        {
+          "name": "query0",
+          "query": "FIND azure_vm WITH disablePasswordAuthentication = true AS v\n  THAT ASSIGNED azure_service_principal\n  THAT ASSIGNED azure_role_assignment WITH actions = '*'\n  THAT ALLOWS azure_subscription as s\nRETURN \n   v.displayName,v.active,v.usesManagedDisks,\n   s.displayName",
```

Comment:
There should be a `USES` relationship between `azure_vm` and `azure_service_principal` instead of an `ASSIGNED` relationship.

I also didn't see an `ALLOWS` relationship between `azure_role_assignment` and `azure_subscription` in the docs.

https://community.askj1.com/kb/articles/1232-azure

![Screen Shot 2022-08-22 at 9 22 25 AM](https://user-images.githubusercontent.com/23426865/185931862-309d4522-0d74-4955-9ff7-2ca82f4d06fe.png)
![Screen Shot 2022-08-22 at 9 21 25 AM](https://user-images.githubusercontent.com/23426865/185931864-dff43d4d-8924-4f2d-8428-8e75522c241d.png)

Reply:
Modified the alert to reflect single property namely disablePasswordAuthentication.
Please review and thanks in advance

Reply:
@jacob-dowdy the alert has been modified to reflect single property. 
Please review TIA